tools/manual/README.md

```diff
@@ -29,3 +29,25 @@ implementation of a tool does not do. It is not a goal that the
 It is not expected that we will be able to implement
 hand-differentiated versions of all evals. AD is after all most useful
 for those cases where hand-differentiation is impractical.
+
+### Multithreading
+
+The primal functions use the [C++ reference
+implementations][../../cpp/gradbench/evals] - see the Commentary for
```

Comment:
Similar to above, this link is broken and should also be replaced with one using a leading slash.